### PR TITLE
Font names, vertical alignment and basic borders support

### PR DESCRIPTION
**Pull request by unti1x**
At first, I had to update deps to make tests work. Everything works good. 

Currently to set borders, you have to set each of them separately. I think it'll do for the first time and should be changed in future, but have no idea how to do it better for now.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,10 @@
 /deps
 erl_crash.dump
 *.ez
+mix.lock
```
Comment (xou) [inline .gitignore]: Unsure that this should be here. Personally, I would say "leave it", but http://elixir-lang.org/getting-started/mix-otp/dependencies-and-umbrella-apps.html says:

> The lock file must be checked in to your version control system, to guarantee that everyone who uses the project will use the same dependency versions as you.

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+
+# Generated by mix hex.docs
+doc
```
Comment (xou) [inline .gitignore]: Duplicate (below)

**Comment by unti1x** [comment]
Fixed

**Comment by xou** [comment]
Awesome, thank you very much!

**Pull request merged**